src/tokens/slice.ts

```diff
@@ -97,10 +97,20 @@ export interface TokenBalancesWithAddress {
   [tokenAddress: string]: TokenBalanceWithAddress | undefined
 }
 
+export interface ImportedToken {
```

Comment:
instead of declaring a new interface that could diverge with the tokens interface, could we use `BaseToken`? or `Omit<BaseToken, ...>`?